### PR TITLE
Update operator-sdk to 0.18

### PR DESCRIPTION
**Pull request by Rulox**
### Proposed changes
Following this guide: https://sdk.operatorframework.io/docs/migration/version-upgrade-guide/

I've updated operator-sdk to `0.18`.

Note: `Makefile` for generating bundle will have to change before the next release, but I'll do that in another PR to prepare the release, because those changes will come when we copy the KIC CRDS here.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -14,7 +14,7 @@ build: binary
 	docker build -f build/Dockerfile -t $(IMAGE):$(TAG) .
 
 run-local:
-	operator-sdk run --local --namespace=$(RUN_NAMESPACE)
+	operator-sdk run local
```
Comment (Dean-Coakley) [inline Makefile:17]: Was this change intentional?

From: https://sdk.operatorframework.io/docs/migration/version-upgrade-guide/#v017x

> The --namespace flag from operator-sdk run --local, operator-sdk test --local, and operator-sdk cleanup was deprecated and is replaced by --watch-namespace and --operator-namespace.
...


I would've expected this change to be:
```suggestion
	operator-sdk run local --operator-namespace=$(RUN_NAMESPACE)
```
(assuming that `--namespace` set the ns where the operator is deployed.

Comment (Dean-Coakley) [reply Makefile:17]: If this was intentional, can `RUN_NAMESPACE` be removed from the Makefile as it is no longer used?

Comment (Rulox) [reply Makefile:17]: We can add  --operator-namespace=$(RUN_NAMESPACE) yeah no prob, I'll add it.

For local development I've experienced that it really doesn't matter, but as u said, then I should have removed the Makefile var
